worker/src/handlers/statement/allowedWorkerQueues.js

```diff
@@ -0,0 +1,51 @@
+import logger from 'lib/logger';
+import {
+  STATEMENT_QUERYBUILDERCACHE_QUEUE,
+  STATEMENT_EXTRACT_PERSONAS_QUEUE,
+  STATEMENT_FORWARDING_QUEUE,
+} from 'lib/constants/statements';
+
+/**
+ * allowable Worker Queues
+ *
+ * @type {Map<string, true>}
+ */
+const allowableWorkerQueues = new Map([
```

Comment:
Could this just be an array of these constants instead of a map to make it simpler?

Reply:
Yes it can be Array. Sorry, this implement was coded before I got to know the allowable queues is just 3. Array.includes is O(N), but Map.has is O(1). Additionally, this function is called every time statement comes. So I used Map for the performance. But size of allowable queues is 3, Array.includes is fast as well. I'll rewrite with array.

Reply:
Yeah appreciate the O(N) vs O(1) what you did makes sense, I don't think the array will get too large. Okay 👍